nestkernel/event.cpp

```diff
@@ -122,5 +122,6 @@ void GapJEvent::operator()()
 }
 
 synindex GapJEvent::synid_ = invalid_synindex;
```

Comment:
I suppose we won't need the `invalid_synindex` any more, we could just check if `synid_vector_` is empty.